templates/zerver/integrations/git.md

```diff
@@ -1,31 +1,41 @@
+# Zulip Git integration
+
 Get Zulip notifications for your Git repositories!
 
+{start_tabs}
+
 1. {!create-an-incoming-webhook.md!}
 
 1. {!download-python-bindings.md!}
 
-1. {!create-channel.md!}
-
 1. {!change-zulip-config-file.md!}
 
-    You may also need to change the value of `STREAM_NAME`.
+    To customize further, refer to the
+    [configuration options](#configuration-options).
 
-    You can specify the branches that will be used for notifications by modifying
```

Comment:
This was previously in a tip, now has been moved into the configuration options section.